packages/connect/src/data/config.ts

```diff
@@ -212,5 +212,11 @@ export const config = {
             methods: ['getFirmwareHash'],
             min: { T1B1: '1.11.1', T2T1: '2.5.1' },
         },
+        {
+            methods: ['solanaGetPublicKey', 'solanaGetAddress', 'solanaSignTransaction'],
+            // TODO solana (vl/connect): set proper version
+            min: { T1B1: '0', T2T1: '2.7.0' },
```

Comment:
I guess, there should be 3 records? T2B1 is imho missing here

Reply:
Updated. I also removed the `comment` prop which was actually copy/pasted and false I think. -> https://github.com/trezor/trezor-suite/pull/9958/commits/5b886ffb3d989c4b945426d52e46f4a472ae0c56#diff-a62b0d8c3182db370268ec303b89e1505852bdd17e2f8f7879fa29df33422fa0R218